services/app-api/emailer/templates.ts

```diff
@@ -149,5 +149,33 @@ const newPackageStateEmail = (
     }
 }
 
-export { newPackageCMSEmail, newPackageStateEmail }
+const unlockPackageCMSEmail = (
+    submission: StateSubmissionType,
+    config: EmailConfiguration
+): EmailData => {
+    const isTestEnvironment = config.stage !== 'prod'
+    const reviewerEmails = config.cmsReviewSharedEmails
+    const bodyHTML = `
+        <h1>Submission ${submissionName(submission)} was unlocked</h1>
```

Comment:
Don't forget the `<span style="color:#FF0000;font-weight:bold;">Note: This submission is part of the MC-Review testing process. This is NOT an official submission and will only be used for testing purposes.</span>` at the top. Maybe we should be sharing this across templates and prepending everything.